code/game/gamemodes/malfunction/newmalf_ability_trees/tree_manipulation.dm

```diff
@@ -152,7 +152,8 @@
 
 	if(!ability_prechecks(user, price))
 		return
-
+	if (alert(user, "Are you sure? This will cause significant damage AND atmospheric breaches. You will be held responsible on an OOC level if this feature is abused.", "Machine Overload:", "Yes", "No") != "Yes")
```

Comment:
I feel like warning this for every instance might be tiresome. Perhaps restrict it to larger things that cause significant damage, like machinery, APCs, SMESs, etc.

Reply:
If you use machine overload in a tasteful way, I don't think it would be tiresome. You can still screw up large portions of the station with one click to an SMES. This will slow down the player and make them think for a brief time.